### DISCUSSION TITLE
Multi-sorting without pressing Shift button

### DISCUSSION THREAD

**Discussion by tqdo** [TanStack/table#2691] [General]
The title says it all. Is there any way to do multi-sorting without having to press Shift button? Thanks

**Comment by meloshnikov** [comment]
> The title says it all. Is there any way to do multi-sorting without having to press Shift button? Thanks

For react-table-v7, the solution is as follows:
`isMultiSortEvent: ((e: React.MouseEvent<Element, MouseEvent>) => boolean)`
...
```
const tableInstance = useTable({
    data,
    columns,
    initialState,
    isMultiSortEvent: () => true,
  }, tableHooks, useGlobalFilter, useSortBy, useRowSelect); 
```

**Reply by neobats** [reply]
If, like me, you found that multi-sorting is frustrating with the shift click, I literally put in the same `() => true` and now it's on click like the regular sorting. So clicking an nth column adds that nth column to the criteria for sorting.
